### ISSUE TITLE
Steps with same names are not updating their status from in progress

### ISSUE THREAD

**Issue by riz-hossain** [AutomationSolutionz/Framework_0.1#380]
![image](https://cloud.githubusercontent.com/assets/5302523/6097955/8a7cd630-af9d-11e4-9044-d0a4cbb347ef.png)
http://demo.automationsolutionz.com/Home/RunID/Sun-Feb-8-14:14:55-2015/

**Comment by Taitalus** [comment]
![production_server](https://cloud.githubusercontent.com/assets/5759182/6101342/cc5af21c-b04c-11e4-843d-ef9a9a0964d4.png)
http://demo.automationsolutionz.com/Home/RunID/Sun-Feb-8-23:58:12-2015/

**Comment by Taitalus** [comment]
Ran the automation run in the production server and selected my machine as the test machine. This is what comes out. I did it every time it is okay. I don't know what kind of sorcery is this.

**Comment by Taitalus** [comment]
Got it. You should always return 'PASSED' instead you returned 'PASS' 
and it always checked in the browser to match it as 'PASSED' and thus though passed it didn't get updated.

**Comment by riz-hossain** [comment]
I thought I created a ticket for it to accept various formats Like passed, Pass, True, and so on...                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the TELUS network.                                                                                                                                                                                                                From: Raju Ahmed ShetuSent: Monday, February 9, 2015 4:51 AMTo: AutomationSolutionz/Framework_0.1Reply To: AutomationSolutionz/Framework_0.1Cc: Riz HossainSubject: Re: [Framework_0.1] Steps with same names are not updating their status from in progress (#380)Got it. You should always return 'PASSED' instead you returned 'PASS' 
and it always checked in the browser to match it as 'PASSED' and thus though passed it didn't get updated. 

—Reply to this email directly or view it on GitHub.

**Comment by Taitalus** [comment]
Yeah. We discussed about it. And you didn't give me the list of values. And eventually we moved on to others topic. Give me the list I will create and implement one.

**Comment by riz-hossain** [comment]
It shouldn't matter about upper or lower case... so convert everything on main driver end to lower case once you get the variable,if it's text.  Pass, passed, True (text), True (boolean) , 1 (integer) , 1 (string) Fail, failed, false (text), false (boolean), 0 (integer), 0 (string)                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the TELUS network.                                                                                                                                                                                                                From: Raju Ahmed ShetuSent: Monday, February 9, 2015 8:38 AMTo: AutomationSolutionz/Framework_0.1Reply To: AutomationSolutionz/Framework_0.1Cc: Riz HossainSubject: Re: [Framework_0.1] Steps with same names are not updating their status from in progress (#380)Yeah. We discussed about it. And you didn't give me the list of values. And eventually we moved on to others topic. Give me the list I will create and implement one.

—Reply to this email directly or view it on GitHub.

**Comment by Taitalus** [comment]
Okay got it.i will implement it right away.                                                                                                                                                                                                                                                                          Raju Ahmed ShetuSoftware DeveloperAutomation Solutionz1212 Countrystone Drive,Kitchener, N2N 3R4.                                                                                                                                                                                                                From: Riz HossainSent: Monday, February 9, 2015 7:44 PMTo: AutomationSolutionz/Framework_0.1Reply To: AutomationSolutionz/Framework_0.1Cc: Raju Ahmed ShetuSubject: Re: [Framework_0.1] Steps with same names are not updating their status from in progress (#380)                                                                                      It shouldn't matter about upper or lower case... so convert everything on main driver end to lower case once you get the variable,if it's text.  Pass, passed, True (text), True (boolean) , 1 (integer) , 1 (string) Fail, failed, false (text), false (boolean), 0 (integer), 0 (string)                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the TELUS network.                                                                                                                                                                                                                From: Raju Ahmed ShetuSent: Monday, February 9, 2015 8:38 AMTo: AutomationSolutionz/Framework_0.1Reply To: AutomationSolutionz/Framework_0.1Cc: Riz HossainSubject: Re: [Framework_0.1] Steps with same names are not updating their status from in progress (#380)Yeah. We discussed about it. And you didn't give me the list of values. And eventually we moved on to others topic. Give me the list I will create and implement one.

—Reply to this email directly or view it on GitHub.

—Reply to this email directly or view it on GitHub.
